src/main.rs

```diff
@@ -76,21 +78,28 @@ fn main() {
         process::exit(0);
     }
 
-    if args.flag_file.is_none() {
+
+    if args.flag_string.is_none() && args.flag_file.is_none() {
         println!("{}", USAGE);
         process::exit(0);
     }
 
-    let file = match read_file(&args.flag_file.unwrap()) {
-        Ok(file) => file,
-        Err(err) => {
-            println!("ERR Couldn't read file: {:?}", err);
-            process::exit(1)
-        }
+
+    let input = match args.flag_string.is_none() {
+        true => {
```

Comment:
Currently, an additional character seems to be necessary in order to prevent a panic with the tokenizer:

````bash
$ badger -e "2**2;"
thread 'main' panicked at 'Unexpected token `Identifier("string")` 16', src/tokenizer.rs:1057
stack backtrace:
   1:     0x5631a24bdd59 - std::sys::backtrace::tracing::imp::write::h00e948915d1e4c72
   2:     0x5631a24c10dc - std::panicking::default_hook::_{{closure}}::h7b8a142818383fb8
   3:     0x5631a24c0389 - std::panicking::default_hook::h41cf296f654245d7
   4:     0x5631a24c0a18 - std::panicking::rust_panic_with_hook::h4cbd7ca63ce1aee9
   5:     0x5631a24c0872 - std::panicking::begin_panic::h93672d0313d5e8e9
   6:     0x5631a24c07e0 - std::panicking::begin_panic_fmt::hd0daa02942245d81
   7:     0x5631a22cebda - badger::tokenizer::Tokenizer::expect_semicolon::hdae7d484c7f80193
                        at /home/fullname/ExternalSource/HoneyBadger/<std macros>:8
   8:     0x5631a22d2f62 - badger::parser::Parser::labeled_or_expression_statement::hb53b13d2bff1b2b9
                        at /home/fullname/ExternalSource/HoneyBadger/src/parser.rs:503
   9:     0x5631a22d6987 - badger::parser::Parser::statement::h0a743db0668da8a2
                        at /home/fullname/ExternalSource/HoneyBadger/src/parser.rs:804
  10:     0x5631a22d6b80 - badger::parser::parse::hb2302add917e17f1
                        at /home/fullname/ExternalSource/HoneyBadger/src/parser.rs:817
  11:     0x5631a22e58d0 - badger::main::hf273a24458a37b12
                        at /home/fullname/ExternalSource/HoneyBadger/src/main.rs:102
  12:     0x5631a24c8bc6 - __rust_maybe_catch_panic
  13:     0x5631a24bfb02 - std::rt::lang_start::h53bf99b0829cc03c
  14:     0x5631a22e8643 - main
  15:     0x7f0781c0482f - __libc_start_main
  16:     0x5631a22a9518 - _start
  17:                0x0 - <unknown>
````

When debugging further, it looks like index exceeds length in the tokenizer.